pkgs/development/libraries/blst/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, lib, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation ( finalAttrs: {
+  pname = "blst";
+  version = "0.3.10";
+
+  src = fetchFromGitHub {
+    owner = "supranational";
+    repo = "blst";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-xero1aTe2v4IhWIJaEDUsVDOfE77dOV5zKeHWntHogY=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+
+    ./build.sh
+
+    runHook postBuild
+  '';
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/lib
+    cp ./libblst.a $out/lib/
+
+    runHook postInstall
+  '';
+
+  enableParallelBuilding = true;
```

Comment:
I don't think that is doing anything when providing a custom buildPhase

Reply:
Should I just remove this line then?

Reply:
Yes, remove it.